tests/ci_build/conda_env/macos_cpu_test.yml

```diff
@@ -32,11 +32,10 @@ dependencies:
 - jsonschema
 - boto3
 - awscli
-- py-ubjson
 - cffi
 - pyarrow
 - pyspark>=3.4.0
 - cloudpickle
 - pip:
   - sphinx_rtd_theme
-  - datatable
+  - py-ubjson
```

Comment:
Is there any difficulty installing it from conda-forge?

Reply:
Yes, Conda-forge doesn't yet offer py-ujson for Apple M1.